drivers/SmartThings/zigbee-contact/src/test/test_smartsense_multi.lua

```diff
@@ -290,6 +290,51 @@ test.register_coroutine_test(
     end
 )
 
+test.register_coroutine_test(
+  "Report from cluster 0xFC03, command 0x09 should be handled as: Temperature (-25.5 C)",
+  function()
+    test.socket.zigbee:__queue_receive({
+      mock_device.id,
+      build_multi_status_report_message(mock_device, "\x01\xFF\x00\x00")
```

Comment:
I'd suggest adding a test or changing one of these tests to also check the handling of 0xFFFF

Reply:
is there something specific about 0xFFFF?

Reply:
It's an edge case

Reply:
handling in 0feff872d5d176eba6cdcec5f8a4e6f3b719c12f

Reply:
Thanks!